azkaban-common/src/main/java/azkaban/executor/ExecutionRampDao.java

```diff
@@ -297,8 +294,29 @@ public void insertAction(final String tableName, Map<String, Object> actionData)
       );
     }
 
-
     try {
+      if ("ramp".equalsIgnoreCase(tableName)) {
```

Comment:
This block will make sure the default value being added when appending record.
Let's say, if the client didn't checkin the startTime field for Ramp Table, the system will auto append the timestamp.

Reply:
Please add appropriate comment to explain the functionality.